Vagrantfile

```diff
@@ -0,0 +1,36 @@
+Vagrant.configure("2") do |config|
+
+  config.vm.define "test" do |test|
```

Comment:
This is cool. Can we combine with normal vagrant up? So we can `vagrant up` to get a VM where we can mess around with the script. And also `vagrant test` to automatically run tests. I'm not too familiar with the Vagrantfile definitions.